eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/processor/SendAsyncMessageProcessor.java

```diff
@@ -310,10 +310,11 @@ public void processRequest(ChannelHandlerContext ctx, AsyncContext<HttpCommand>
 
                     @Override
                     public void onSuccess(SendResult sendResult) {
+                        StringBuilder sb = new StringBuilder();
                         HttpCommand succ = request.createHttpCommandResponse(
                             sendMessageResponseHeader,
                             SendMessageResponseBody.buildBody(EventMeshRetCode.SUCCESS.getRetCode(),
-                                EventMeshRetCode.SUCCESS.getErrMsg() + sendResult.toString()));
+                                EventMeshRetCode.SUCCESS.getErrMsg() + sb.append(sendResult.toString())));
```

Comment:
This issue is mainly to solve the NPE generated by the .toString() method, and your change like this does not seem to solve the problem. So I suggest just removing .toString(), what do you think?